### ISSUE TITLE
F523 / F524 false positive

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#745]
In GitLab by @glitchov on May 27, 2020, 23:44

F523 '...'.format(...) has unused arguments at position(s): 0

F524 '...'.format(...) is missing argument(s) for placeholder(s): a, b, c

The code is:

def f(x, y, **kw):

    print("{a}:{b}:{c}".format(**kw))

*************************************************************************
*Please describe how you installed Flake8*

pip install pre-commit

and it came at the first run

**Comment by asottile** [comment]
In GitLab by @asottile on May 27, 2020, 23:57

from the issue template:

```
<!--
*************************************************************************
NOTE: flake8 is a linting framework and does not implement any checks

if you are reporting a problem with a particular check, please track down
the plugin which implements that check.

some common ones:
- F###: https://github.com/pycqa/pyflakes
- E###, W###: https://github.com/pycqa/pycodestyle
*************************************************************************
-->
```

**Comment by asottile** [comment]
In GitLab by @asottile on May 27, 2020, 23:57

closed

**Comment by asottile** [comment]
In GitLab by @glitchov on May 28, 2020, 24:00

As the message begins with F i guess it's pyflakes
